aws_iam_group_policy.tf

```diff
@@ -3,20 +3,7 @@
 #}
 
 resource "aws_iam_group_policy" "iamGrpPolicyWithFullAdminCntrl" {
-  name = "test"
-  group = "test"
```

Comment:
**Accurics remediation**
 
**[HIGH]: Ensure IAM policies that allow full administrative privileges are not created**
 
**Vulnerability**
It is recommended and considered a standard security advice to grant least privileges that is, granting only the permissions required to perform a task. IAM policies are the means by which privileges are granted to users, groups, or roles. Determine what users need to do and then craft policies for them that let the users perform only those tasks, instead of granting full administrative privileges.
 
**Remediation**
For each account that failed this rule, on the IAM console. Navigate to Policies. Search for the policy that failed the rule. Rework the permissions in the policy to grant positive permissions to specific AWS services or actions instead of blanket permissions using '*'. For remediation, refer to control 1.22 in the CIS Amazon Web Services Foundations Benchmark v1.1.0 document that can be downloaded from: https://downloads.cisecurity.org/#/all. Additional Reference: CIS Amazon Web Services Foundations Benchmark v1.1.2 

source: accurics-